src/main/java/duke/UI.java

```diff
@@ -0,0 +1,101 @@
+package duke;
+
+/**
+ * UI class to abstract all UI related printing.
+ */
+public class UI {
+
+    /**
+     * Commonly used beautifying line.
+     */
+    public static String line = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
```

Comment:
Please be reminded that the names of constants should be in all uppercase (and use underscores between separate words)